apps/dev-portal-web/docs/api/examples/auth.en.mdx

```diff
@@ -43,6 +45,37 @@ curl 'https://condo.d.doma.ai/admin/api' -H 'Content-Type: application/json' --d
 ```
 </Tabs>
 
+## authenticateUserWithPassword
+
+Authentication with email and password may be used for service users. It works the same way as `authenticateUserWithPhoneAndPassword`.
+
+<Tabs labels={['GQL-query example', 'CURL-query example', 'Response example']}>
+    ```graphql
+    mutation {
+      authenticateUserWithPassword(email: "e@ma.il" password: "***") {
+        token item { id name }
+      }
+    }
+    ```
+    ```bash
+    curl 'https://condo.d.doma.ai/admin/api' -H 'Content-Type: application/json' --data-raw $'{"operationName":"authenticateUserWithPassword","variables":{"identity":"e@ma.il","password":"12345678"},"query":"mutation authenticateUserWithPassword($identity: String!, $password: String!) { obj: authenticateUserWithPassword(email: $identity, password: $password) { token item { id name } } }"}'
+    ```
+
+    ```json
+    {
+      "data": {
+        "authenticateUserWithPassword": {
+          "token": "lSULdzVaPWBK8CTH-7A5JpL8YFRk3YNg0.8R+N+OrJXoAA7fNe60EBR+ckkQ73SSZqa/7WXTTG6ZMMQ",
```

Comment:
Make it more fake, like 123123213asdasd.123123

Reply:
Just copied from example above. But ok, fixed.